### DISCUSSION TITLE
정말 사소한 건데

### DISCUSSION THREAD

**Discussion by keelim** [tony9402/baekjoon#198] [Question & Answer] [answered]
좋은 레포지토리 만들어주셔서 감사합니다.

정말 사소한 건데 문자열 파트에서 
![image](https://user-images.githubusercontent.com/26667456/124612868-a55d5d80-dead-11eb-898a-0f8f0894b466.png)

제대로 이미지가 링킹이 동작안하는 것 같습니다.  몇몇 부분 있는 것 같은데 수정해서 풀리퀘 해도  될까요?

다시 한번 좋은 레포 정말 감사드립니다.👍

**Comment by tony9402** [comment, marked as answer]
ㅎㅎ 감사합니다.. !

해당 오류는 이미지가 많아서 그런지 가끔 이미지가 제대로 로딩이 안될때가 있는거 같습니다 !
혹시 더 좋은 방법이 있다면 알려주시면 감사하겠습니다 :)

**Reply by keelim** [reply]
아 그렇군여  github 에서는 로딩이 오래 걸리는 건 생각도 안해봤는데 
답변 감사합니다. 
좋은 하루 보내세요!!!
